### DISCUSSION TITLE
How to calculate the number of supported cameras with Tensorflow GPU

### DISCUSSION THREAD

**Discussion by wxt** [blakeblackshear/frigate#7560] [General]
I am trying to get a Nvidia GPU for both decoding and Tensorflow for the object detection.  Is there a rough estimate or some form of calculation on how many cameras can a specific GPU support? From what I've read so far, the number of expected motion detected and objects matters a lot?

My place is a suburb corner lot with 5 1080p cameras.  1 faces the front door and other 4 faces each side of the house.  What would be the lowest spec GPU that can support those cameras

**Comment by NickM-27** [comment]
You need to look at the decode performance of the GPU as well as inference time. Motion detection is not run on the GPU. 

The docs have a handful of examples of GPUs users have used.
